docs/content/advanced/backend/entities/index.md

```diff
@@ -2,19 +2,6 @@
 
 In this document, you’ll learn how you can create an [Entity](overview.md).
 
```

Comment:
Can you remove this empty line? There are now two empty lines above the "Create the Entity" headline.

Reply:
i have removed the empty lines . can you please check now